features/collection.feature

```diff
@@ -0,0 +1,9 @@
+Feature:
+  In order to use Postman with API Platform
+  as developper
+  then i should be able to generate a Postman collection file
+
+  Scenario: Generates a Postman collection with 1 folder and 5 requests
+    When I generate the Postman collection
+    Then I should have 1 folder in Postman collection
+    And I should have 5 requests in Postman collection
```

Comment:
Maybe you should add a step to ensure there is a correct number of requests on each folder, like:
```gherkin
And I should have 5 requests in Postman collection in folder Foo
```
Also, to check requests are correctly set on related folder, you could check generated JSON, like:
```gherkin
And folder Foo should be like:
"""
{
    "id": "azert-1234",
    "name": "Foo",
    "requests": [
        "fghjk-2345",
        "lhggfgfd-976"
    ]
}
"""
```